xbmc/addons/ContextMenuAddon.cpp

```diff
@@ -87,10 +87,17 @@ void CContextMenuAddon::ParseMenu(
     if (StringUtils::IsNaturalNumber(label))
       label = g_localizeStrings.GetAddonString(ID(), atoi(label.c_str()));
 
+    std::vector<std::string> args;
+    args.push_back(ID());
+ 
+    std::string arg = element.second.GetValue("@args").asString();
+    if (!args.empty())
```

Comment:
```suggestion
    if (!arg.empty())
```

Reply:
Just found that one. I also update the `==` operator.